STARTER/STARTER.libraries.yml

```diff
@@ -0,0 +1,10 @@
+global:
+  version: VERSION
+  css:
+    theme:
+      css/style.css: {}
+  js:
+    js/dist/themekit.js: {}
```

Comment:
I don't like `themekit` here, maybe `draft`. Also, needs to be changed in gulpfile